docs/update-prod-data.md

```diff
@@ -0,0 +1,82 @@
+# Steps to update the dev db with prod data
+
+
+## 1 - Take a Dump of Data from Production and Export to a File
+
+You'll first need to take a dump of the data from the production environment and export it to a file. This file will contain all of the data that is currently in the production database.
+
+You will first have to use **kubectl** to get the list of available pods `kubectl get pod`
+Locate the one with the prefix `lists-postgres- `
+
+The you'll have to access the shell of that pod with the following command:
+```bash
+kubectl exec -it lists-postgres-[pod_id]
+```
+
+_Replace [pod_id] with the reelevant id, eg `764c9b5879-5mf5f`_
+
+Now you can run the following command:
+
+```bash
+pg_dump -U master -h lists-postgres:5432 -Fc lists > prod_dump.dump
+```
+
+## 2 - Load the Dump into a Local Development Database
+
+Once you have the dump file, you'll need to load it into a local development database so you can make changes without affecting the production environment.
+
+Here's an example of how to load a Postgres dump file into a local development database:
+
+```bash
+psql -U master -h localhost -d lists < prod_data.sql
+```
+
+
+## 3 - Update Emails on All Lists in Local Dev Database
+
+Now that you have the data from the production database loaded into your local development database, you can make the necessary changes. In this case, you need to update the emails on all of the lists in the database.
```

Comment:
filler 
(I know that I have the data from the production database loaded because I've just completed steps 1 and 2).